scheduler/pkg/kafka/gateway/infer.go

```diff
@@ -194,7 +196,7 @@ func (kc *InferKafkaHandler) GetNumModels() int {
 func (kc *InferKafkaHandler) createTopics(topicNames []string) error {
 	logger := kc.logger.WithField("func", "createTopics")
 	if kc.adminClient == nil {
-		logger.Warnf("Can't create topics %v as no admin client", topicNames)
+		logger.Warnf("no kafka admin client, can't create any of the following topics: %v", topicNames)
```

Comment:
return error?

Reply:
It appears that returning an error here fails the go tests